### DISCUSSION TITLE
coc.nvim Error

### DISCUSSION THREAD

**Discussion by JeanOsorio** [NvChad/NvChad#1347] [General]
Hello guys I don't want to open an issue because I have another machine where nvchad works perfect. So here is my problem, when I try to install and open NVChad I get the following error:

<img width="1680" alt="Screenshot 2022-07-18 at 12 26 04" src="https://user-images.githubusercontent.com/1721572/179492827-c21e44f9-c3be-410a-bdab-0a9bdf4c5db8.png">

I have python 2.7.18 and pythong 3.7.9 installed on my machine, I also install pynvim but still getting the error. Any suggestion or idea because I find anything related to this problem.

**Comment by siduck** [comment]
we dont use coc so not sure why you get this error

**Reply by JeanOsorio** [reply]
I found a coc folder inside my .config folder and after I delete the folder everything start to work.

**Reply by siduck** [reply]
oh, its better to delete or backup previous nvim config then install nvchad

**Reply by martin-braun** [reply]
@siduck I understand that CoC is not really supported, but it should be possible to integrate. What speaks against CoC? Are there any docs that can help to get CoC working with NvChad?

**Reply by siduck** [reply]
@martin-braun Just remove cmp + its related helper plugins & lspconfig and install coc.nvim :)

**Reply by martin-braun** [reply]
@siduck I will give it a try when I'm back at work. Thank you. So cmp is for suggestions and lsp for language support. CoC is a substitute to both, because it has suggestions and language servers. 

I will give it a try.
